config/src/config/network_config.rs

```diff
@@ -35,17 +35,17 @@ pub struct NetworkConfig {
     pub discovery_interval_ms: u64,
     pub connectivity_check_interval_ms: u64,
     // Flag to toggle if Noise is used for encryption and authentication.
-    pub enable_encryption_and_authentication: bool,
-    // If the network is permissioned, only trusted peers are allowed to connect. Otherwise, any
-    // node can connect. If this flag is set to true, the `enable_encryption_and_authentication`
+    pub enable_noise: bool,
+    // If network authentication is enabled, only trusted peers are allowed to connect. Otherwise, any
+    // node can connect. If this flag is set to true, the `enable_noise`
```

Comment:
the should be removed before `enable_noise`

Reply:
Done, thanks!